src/__tests__/stripeObject.test.js

```diff
@@ -0,0 +1,106 @@
+const StripeObject = require("../StripeObject");
```

Comment:
This fails for me on Linux with a case-sensitive filesystem. `../stripeObject` works fine.

Reply:
Yeah the problem here is that I want to rename stripeObject.js to StripeObject.js to be consistent with class file naming conventions. But for some reason git doesn't pick up the change. I might have to delete the files, commit that deletion, then re-add them so that they're changed. Let me try that!